crates/common/consensus/lean/src/lib.rs

```diff
@@ -117,7 +44,7 @@ pub fn get_latest_justified_hash(post_states: &HashMap<B256, LeanState>) -> Opti
 pub fn get_fork_choice_head(
     blocks: &HashMap<B256, Block>,
     provided_root: &B256,
-    votes: &[Vote],
+    votes: &Vec<SignedVote>,
```

Comment:
```suggestion
    votes: &[SignedVote],
```

we shouldn't be passing references to a vector, we should just pass a slice assuming there are no stack problems due to the underlying datatypes